src/Controller/Admin/BoardCrudController.php

```diff
@@ -58,8 +69,14 @@ public function configureFields(string $pageName): iterable
         yield IntegerField::new('hearingResponseDeadline', 'Hearing response deadline(days)');
         yield IntegerField::new('finishHearingDeadlineDefault', 'Finish hearing deadline(days)');
         yield IntegerField::new('finishProcessingDeadlineDefault', 'Finish processing case deadline(days)');
-        yield TextareaField::new('complainantPartyTypes', 'Complainant party types');
-        yield TextareaField::new('counterPartyTypes', 'Counter party types');
-        yield TextareaField::new('statuses', 'Statuses');
+        yield TextareaField::new('complainantPartyTypes', 'Complainant party types')
+            ->setHelp($this->translator->trans('Configuration of complainant types. Each line will be treated as a type of which the first will be the one used for sorting purposes.', [], 'admin'))
+        ;
+        yield TextareaField::new('counterPartyTypes', 'Counter party types')
+            ->setHelp($this->translator->trans('Configuration of counterpart types. Each line will be treated as a type of which the first will be the one used for sorting purposes.', [], 'admin'))
```

Comment:
Is it “counter party” og “counterpart”?

Reply:
Good question, most translations of "modpart" to english says "counterparty", so we will go with that.

Reworded help and also renamed helper functions and properties.